src/custom-elements/parts-kit.tsx

```diff
@@ -1,31 +1,86 @@
 import { render } from 'preact'
 import { App } from '../app.tsx'
+// https://vitejs.dev/guide/features.html#disabling-css-injection-into-the-page
+import cssString from '../index.css?inline'
```

Comment:
`?inline` this turns `index.css` into a string that we can insert into the shadow DOM stylesheet.

Reply:
awesome!